src/amqp/amqpsender.h

```diff
@@ -15,8 +15,7 @@
 class AMQPSender : public proton::messaging_handler {
   private:
     std::deque<proton::message> messages_; 
-    proton::url url_;
-    proton::connection conn_;
+    std::string url_;
```

Comment:
This now doesn't validate the URL when the `AMQPSender` is constructed. Is that a regression in error reporting behaviour? What happens after this PR when an invalid URL is given?